assets/translations/en-us.yaml

```diff
@@ -1228,6 +1228,11 @@ cluster:
           }
   memberRoles:
     removeMessage: 'Note: Removing a user will not remove their project permissions'
+    addClusterMember:
+      labelSelect: Select Member
+      labelAdd: Add Member
+      placeholder: Search for a member to provide cluster access
+      searchPlaceholder: Start typing to search for principals
```

Comment:
I think this should just be `Start typing to search`. We're references `member` in the component and as there's a couple of them in the two lines above this can be kept simple